src/bindings/python/src/openvino/utils.py

```diff
@@ -28,3 +29,27 @@ def add_openvino_libs_to_path() -> None:
                 # On Windows, with Python >= 3.8, DLLs are no longer imported from the PATH.
                 if (3, 8) <= sys.version_info:
                     os.add_dll_directory(os.path.abspath(lib_path))
+
+
+def deprecated(version: str = "", message: str = "") -> Callable[..., Any]:
```

Comment:
This decorator seems a bit heavy, wouldn't something like this be sufficient?
```suggestion
def deprecated(wrapped: Callable[..., Any], version: str = "", message: str = "") -> Callable[..., Any]:
    """Prints deprecation warning and runs the function.
    :param wrapped: The function to decorate
    :param version: The version in which the code will be removed.
    :param message: A message explaining why the function is deprecated and/or what to use instead.
    """
    @wraps(wrapped)
    def wrapper(*args: Any, **kwargs: Any):
        deprecation_warning(wrapped.__name__, version, message)
        return wrapped(*args, **kwargs)
    return wrapper
```

Reply:
Nope. It cannot be like that. It won't work if you add the 'wrapped' argument to the function up:
```Traceback (most recent call last):
  File "/home/adrian/PycharmProjects/pythonProject/main.py", line 20, in <module>
    @deprecated(message="msg")
TypeError: deprecated() missing 1 required positional argument: 'wrapped'```

Reply:
see Example #3 here: https://www.geeksforgeeks.org/decorators-with-parameters-in-python/